main.css

```diff
@@ -36,7 +36,6 @@ code {
 div.body {
   margin: 2em auto;
   max-width: 38.75em;
-  padding: 1em;
```

Comment:
What's the reason for removing this?

Reply:
There's a lot of whitespace at the top of each webpage which makes the web page look weird.